secret_number.js

```diff
@@ -1,4 +1,11 @@
-'use strict';
+"use strict";
 module.exports = function() {
-  
-};
\ No newline at end of file
+  let secretNumberGenerator = function() {
```

Comment:
So this doesn't need to be in a function since you're just calling it right after you declare the function. You could streamline the code by just doing: 
```
let secretNumber = Math.floor(Math.random() * 1000000 + 1);
```